### PR TITLE
[node-bridge]: remove API Gateway normalization

### PR DESCRIPTION
**Pull request by Kikobeats**
We no longer use since a long time ago

### CODE REVIEW

File: packages/node-bridge/bridge.js [hunk 1]
```diff
@@ -83,40 +83,9 @@ function normalizeProxyEvent(event) {
...
-    responseCallbackCipherKey: undefined,
-    responseCallbackStream: undefined,
-    responseCallbackUrl: undefined,
-  };
-}
-
 /**
  * @param {import('./types').VercelProxyEvent | import('aws-lambda').APIGatewayProxyEvent} event
- * @return {import('./types').VercelProxyRequest}
+ * @return {import('./types').VercelProxyRequest | undefined }
```
Comment (Kikobeats) [inline packages/node-bridge/bridge.js]: maybe should the method throw an error if `event.Action` is `undefined`?

Comment (TooTallNate) [reply packages/node-bridge/bridge.js]: Ya I think so

File: packages/node-bridge/bridge.js [hunk 2]
```diff
@@ -83,40 +83,9 @@ function normalizeProxyEvent(event) {
...
-    responseCallbackCipher: undefined,
-    responseCallbackCipherIV: undefined,
-    responseCallbackCipherKey: undefined,
-    responseCallbackStream: undefined,
-    responseCallbackUrl: undefined,
-  };
-}
-
 /**
  * @param {import('./types').VercelProxyEvent | import('aws-lambda').APIGatewayProxyEvent} event
```
Comment (TooTallNate) [inline packages/node-bridge/bridge.js]:
```suggestion
 * @param {import('./types').VercelProxyEvent} event
```

**Review by styfle — APPROVED** [review]
I think this is no longer used, but you might want to try it on staging to be sure

**Comment by styfle** [comment]
Technically this should be `semver: major`

**Review by ijjk — APPROVED** [review]

**Pull request merged**